stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -101,6 +102,16 @@ class Value():
     def __init__(self, typecode_or_type: str, *args: Any, lock: bool = ...) -> None: ...
 
 # ----- multiprocessing function stubs -----
+def active_children() -> List[Process]: ...
+def allow_connection_pickling() -> None: ...
 def cpu_count() -> int: ...
 def freeze_support() -> None: ...
+def get_all_start_methods() -> List[str]: ...
+def get_context(method: Optional[str] = ...) -> BaseContext: ...
+def get_logger() -> Logger: ...
+def get_start_method(allow_none: Optional[bool] = ...) -> str: ...
+def log_to_stderr(level: Optional[str] = ...) -> Logger: ...
 def Manager() -> SyncManager: ...
+def set_executable(executable: str) -> None: ...
+def set_forkserver_preload(module_names: List[str]) -> None: ...
+def set_start_method(method: Optional[str] = ...) -> None: ...
```

Comment:
Thanks! On the one hand, it seems that `method` argument is optional and the undocumented `force` one is not present just in `set_start_method` of ` BaseContext` in 3.4. On the other hand, it is overridden (with `method` not optional and the `force` optional) in `DefaultContext`, which seems to contain the imported method from `__init__.py`. Anyway, this is solved >=3.5 and both the latter signature.

Should stub for `set_start_method` of ` BaseContext` in `multiprocessing/context.pyi` (line 94) be corrected with these changes? I mean to be like `set_start_method` of ` DefaultContext` but for >=3.5.